src/storybook/decorators.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Frame } from 'components/common/Frame'
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types'
+
+export const CenteredAndFramed = (DecoratedStory: () => StoryFnReactReturnType): JSX.Element => (
+  <div style={{ textAlign: 'center' }}>
+    <Frame style={{ display: 'inline-block' }}>{DecoratedStory()}</Frame>
```

Comment:
While this is ok, it only centers horizontally. Why not go for a `flex` like we do everywhere else

Reply:
or `inline-flex`

Reply:
Cause I didn't mean to center it vertically :) 

this is just when we want this frame:
![image](https://user-images.githubusercontent.com/2352112/92941661-f7318480-f450-11ea-8589-8d314da9cff1.png)


On top that I would prefer to see it in the top, If we center vertically probably the actions can easily be hiding part of the component.

I'll merge it, let me know if u are still not convinced, I can easily make a PR to address it after.